### PR TITLE
kafka-integration part 1

### PR DESCRIPTION
**Pull request by safwanislam**
WIP: Creating a new pull request since I ran into rebasing issues with old branch (feature/kafka-integration).

### CODE REVIEW

File: log-collector/src/main/scala/io/phdata/pulse/logcollector/LogCollector.scala [hunk 1]
```diff
@@ -53,42 +57,64 @@ object LogCollector extends LazyLogging {
...
       }
+      case _ => {
+        http(cliParser.port(), routes)
+      }
+    }
+  }
 
-    val s = start(cliParser.port())
+  def http(port: Int, routes: LogCollectorRoutes): Future[Unit] = {
+    // Akka System
```
Comment (afoerster) [inline log-collector/src/main/scala/io/phdata/pulse/logcollector/LogCollector.scala]: Nit: unnecessary/inaccurate comment

File: log-collector/src/main/scala/io/phdata/pulse/logcollector/PulseKafkaConsumer.scala [hunk 2]
```diff
@@ -48,8 +45,7 @@ class PulseKafkaConsumer(solrCloudStream: SolrCloudStream) extends JsonSupport w
         val records = consumer.poll(MAX_TIMEOUT)
         for (record <- records.asScala) {
           logger.trace("KAFKA: Consuming " + record.value() + " from topic: " + topic)
-          val logEvent    = record.value().parseJson.convertTo[LogEvent]
-          val logEventMap = Util.logEventToFlattenedMap(logEvent)
+          val logEventMap = record.value().parseJson.convertTo[Map[String, String]]
           solrCloudStream.put(logEventMap.getOrElse("application", ""), logEventMap)
```
Comment (afoerster) [inline log-collector/src/main/scala/io/phdata/pulse/logcollector/PulseKafkaConsumer.scala]: I don't think the 'orElse' here is valid. If there's no application name it's an invalid request and we should just throw it away. For now can you just add a TODO for proper error handling when there is no application name?

Comment (safwanislam) [reply log-collector/src/main/scala/io/phdata/pulse/logcollector/PulseKafkaConsumer.scala]: @afoerster Where should I add the TODO? In Issue #99 ?

Comment (afoerster) [reply log-collector/src/main/scala/io/phdata/pulse/logcollector/PulseKafkaConsumer.scala]: I meant in the code, like // TODO ... (which can be searched easily), but actually adding it to the issue is a good idea

File: log-collector/src/main/scala/io/phdata/pulse/logcollector/LogCollector.scala (hunk 1 above)
Comment (safwanislam) [reply log-collector/src/main/scala/io/phdata/pulse/logcollector/LogCollector.scala]: removed

File: log-collector/src/main/scala/io/phdata/pulse/logcollector/PulseKafkaConsumer.scala (hunk 2 above)
Comment (safwanislam) [reply log-collector/src/main/scala/io/phdata/pulse/logcollector/PulseKafkaConsumer.scala]: added TODO in code and in Issue #99

**Pull request merged**